ts/hooks/useStatusAlertProps.tsx

```diff
@@ -68,6 +77,10 @@ export const useStatusAlertProps = (
   const localeFallback = fallbackForLocalizedMessageKeys(locale);
 
   useEffect(() => {
+    // If the user is offline and the current route is not in the blacklist, show the alert
+    if (blackListOfflineAlertRoutes.has(currentRoute)) {
```

Comment:
This will return before the `connectivityAlert` update, so when navigating to the blacklisted route the activity alert shown is the previous. To effectively remove the alert we should call `setConnectivityAlert(undefined)` and then return. What do you think?

Reply:
Good point! addressed in [9e6b7d4](https://github.com/pagopa/io-app/pull/7008/commits/9e6b7d4b0d725c4782841786f23661fad7d2f4ed)